src/extension.ts

```diff
@@ -31,9 +32,14 @@ function createInterface(args: any) {
     promptAndSave(args, 'interface');
 }
 
-function promptAndSave(args: any, templatetype: string) {
-    if (args == null) args = { _fsPath: vscode.workspace.rootPath }
+function createEnum(args) {
```

Comment:
The compiler has issues with missing the type here.
```suggestion
function createEnum(args: any) {
```